### ISSUE TITLE
[CAL-425] New workflow action to send Whatsapp message

### ISSUE THREAD

**Issue by facinick** [calcom/cal.com#3664]
### Is your proposal related to a problem?

No.

### Describe the solution you'd like

I want to send confirmation mails with video links to attendees over whatsapp, instead of their mails.

### Describe alternatives you've considered

[Calenderly.com](http://Calenderly.com) has workflows that allow me to do this.

### Additional context

(Write your answer here.)

> From [SyncLinear.com](https://synclinear.com)

**Comment by Udit-takkar** [comment]
+1 whatsapp is also very commonly used to send meet links in India.

**Comment by PeerRich** [comment]
that would be a great option in workflows? @CarinaWolli 

- send Whatsapp message to attendee
- send Whatsapp message to specific number

<img width="397" alt="CleanShot 2022-08-03 at 14 30 57@2x" src="https://user-images.githubusercontent.com/8019099/182608183-e9e1d45b-f0d5-4ff6-b134-584a7f24a5cc.png">

for meeting links, do you want a "WhatsApp" option here? those would be two different tickets, feel free to open a new one
<img width="602" alt="CleanShot 2022-08-03 at 14 31 50@2x" src="https://user-images.githubusercontent.com/8019099/182608375-e4efc0ec-0d23-4558-aff2-4a872b47fafa.png">

**Comment by PeerRich** [comment]
@Udit-takkar would someone use whatsapp to send a link (zoom) or would they use whatsapp video?

**Comment by Udit-takkar** [comment]
@PeerRich  although i have not used zoom much but yeah whatsapp  is like most active and engaging messaging app and also used for audio calls.  never used whatsapp video.

**Comment by PeerRich** [comment]
> @PeerRich  although i have not used zoom much but yeah whatsapp  is like most active and engaging messaging app and also used for audio calls.  never used whatsapp video.

wait so what you wanna do is you make an appearance and then chat? i am confused

**Comment by Udit-takkar** [comment]
I mean whatsapp is used a lot in communication so people do share meeting links/ webinar link to each other. even by my college professors on group chats.

**Comment by PeerRich** [comment]
right but they share the linnk to the video service, no? or do they share the link to the whatsapp group?

**Comment by Udit-takkar** [comment]
> right but they share the linnk to the video service, no? or do they share the link to the whatsapp group?
they share the link (meet/zoom) in whatsapp group.  i don't understand what you mean by share the link to video service.

**Comment by CarinaWolli** [comment]
> that would be a great option in workflows https://app.cal.com/workflows? @CarinaWolli
> 
> * send Whatsapp message to attendee
> * send Whatsapp message to specific number
> 
> <img alt="CleanShot 2022-08-03 at 14 30 57@2x" width="397" src="https://user-images.githubusercontent.com/8019099/182608183-e9e1d45b-f0d5-4ff6-b134-584a7f24a5cc.png">

@Udit-takkar As far as I understand a new action for sending a WhatsApp message to the attendee would solve your issue, right? This would allow you to add your personal google meet or zoom link in the custom template and send it via WhatsApp to an attendee once an event is booked

**Comment by Udit-takkar** [comment]
Yeah

**Comment by jemiluv8** [comment]
I'm working on this.

@PeerRich I wonder if all translations for locales have to be "manually" set for all supported languages. I just started with english to get me going for the time being.

I decided to add 2 New workflow actions
- WHATSAPP_ATTENDEE
- [WHATSAPP_NUMBER

<img width="743" alt="Screenshot 2023-04-20 at 10 47 44 PM" src="https://user-images.githubusercontent.com/119384208/233503144-3f1845c8-5567-4cac-b1b0-328a73dcd3fa.png">
